en/docs/administer/monitoring-logs.md

```diff
@@ -59,39 +53,45 @@ See the following information on configuring **Carbon logs**, **Audit
 logs,** **HTTP access logs**, and **Service/Event logs** for WSO2 Identity Server.
 
 #### Configuring Carbon logs 
-You can easily configure Carbon logs using the WSO2 Identity Server Management Console or you can manually edit the
+Carbon logs are enabled in WSO2 Identity Server by default. You can change
+the following default configuration by manually updating the the
```

Comment:
the word "the" has been repeated twice in this sentence. let's remove one "the" please

Reply:
done